Sources/BSWInterfaceKit/SwiftUI/ViewModifiers/InAppNotificationModifier.swift

```diff
@@ -0,0 +1,80 @@
+#if canImport(UIKit.UIViewController)
+
+import SwiftUI
+
+@available(iOS 17, macOS 14, *)
+#Preview {
+    @Previewable
+    @State
+    var state: String? = nil
+    
+    Button {
+        state = "HALA MADRID!"
+    } label: {
+        Text("Marc es muy del Madrid")
+    }
+    .inAppNotification(message: $state, textColor: .white, backgroundColor: .orange)
+}
+
+public extension SwiftUI.View {
+    
+    /// Presents an in-app notification as an overlay on the current view.
+    /// The notification displays a message with customizable text color and background color.
+    /// - Parameters:
+    ///   - message: A binding to a string that triggers the notification when set. Setting it to `nil` dismisses the notification.
+    ///   - textColor: The color of the notification's text. Defaults to `.white`.
+    ///   - backgroundColor: The color of the notification's background. Defaults to `.green`.
+    func inAppNotification(
+        message: Binding<String?>,
```

Comment:
And maybe we should use AttributedString instead of String so we dont have that extra parameter